mainnet/proposals/0/v1.2.0-mainnet.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
@sherry-x Runtian is using different version numbers here to identify framework proposals. Does this make sense to you? This is different from the underlying version tagged in the branch (v1.2.5?)

Reply:
yeah I think framework should version and tag separately, currently we have `aptos-node-v1.2.x`, maybe for framework we can do `aptos-framework-v1.2.0` do we need to add the network name in it? up to you two to decide